src/refiners/foundationals/dinov2/__init__.py

```diff
@@ -0,0 +1,15 @@
+from .dinov2 import (
+    DINOv2_base,
+    DINOv2_base_reg,
+    DINOv2_large,
+    DINOv2_large_reg,
+    DINOv2_small,
+    DINOv2_small_reg,
+)
+from .vit import (
+    ViT,
+    ViT_base,
+    ViT_large,
+    ViT_small,
+    ViT_tiny,
+)
```

Comment:
```suggestion
)

__all__ = [
    "DINOv2_base",
    "DINOv2_base_reg",
    "DINOv2_large",
    "DINOv2_large_reg",
    "DINOv2_small",
    "DINOv2_small_reg",
    "ViT",
    "ViT_base",
    "ViT_large",
    "ViT_small",
    "ViT_tiny",
]

```

Reply:
You need to add this, otherwise, pyright will complain about not accessed variables.